src/main.rs

```diff
@@ -233,6 +494,15 @@ async fn serve_req(
     }
 }
 
+/// iss=triagebot, sub=gh username, uid=gh user_id, exp=now+30', checksum=sha256(user data)
+fn create_cookie_content(user_login: &str, user_id: i64) -> serde_json::Value {
+    let auth_secret = std::env::var("BACKOFFICE_SECRET").expect("BACKOFFICE_SECRET is not set");
+    let exp = Utc::now() + Duration::minutes(30);
+    let digest = format!("{};{};{}", user_id, user_login, auth_secret);
+    let digest = hex_digest(Algorithm::SHA256, &digest.into_bytes());
+    serde_json::json!({"iss":"triagebot", "sub":user_login, "uid": user_id, "exp":exp, "checksum":digest})
+}
```

Comment:
Everytime users log in, a cookie with this content will be saved on their browser. If the cookie is not found, the user will be redirected to the GitHub auth login workflow again